aggregate/clean_aggregated.py

```diff
@@ -37,3 +37,14 @@ def order_PUMS_QOL_multiple_years(categories, measures, years):
                     rv.append(f"{c}_{r}{y}{m}")
 
     return rv
+
+def order_affordable(measures, income) -> List:
```

Comment:
this looks great, I think order affordable should have it's own function for sure